mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -1,27 +1,28 @@
 use tracing::info;
 
-use crate::settings::Settings;
+use crate::settings::ConnectionSettings;
 
-/// Bridge implementation for nested scenario.
-/// It is used when `IOTEDGE_GATEWAYHOSTNAME` env variable is set.
-pub struct NestedBridge {
-    settings: Settings,
+/// Bridge implementation that connects to local broker and remote broker and handles messages flow
+pub struct Bridge {
+    connection_settings: ConnectionSettings,
 }
 
-impl NestedBridge {
-    pub fn new(settings: Settings) -> Self {
-        NestedBridge { settings }
+impl Bridge {
+    pub fn new(connection_settings: ConnectionSettings) -> Self {
+        Bridge {
+            connection_settings,
+        }
     }
 
     pub async fn start(&self) {
-        info!("Starting nested bridge...{:?}", self.settings);
+        info!("Starting bridge...{:?}", self.connection_settings.name());
```

Comment:
I know there are concerns about leaking credentials, but what we can do is have a to_string() method that only returns the non-private fields.